.eslintrc.json

```diff
@@ -2,7 +2,7 @@
     "env": {
         "es2020": true,
         "node": true,
-        "mocha": true
+        "jest": true
     },
     "extends": ["eslint:recommended", "prettier", "plugin:unicorn/recommended"],
```

Comment:
at some point we should introduce `plugin:@typescript-eslint/recommended` but it'll need its own rather large cleanup PR i think since i suspect there's a fair amount of recommended rules we don't yet follow (but should)

Reply:
That's done as an override below

Reply:
i expected a lot more fixing up to get those rules passing, nice work